.rubocop_todo.yml

```diff
@@ -99,9 +94,7 @@ Metrics/BlockLength:
 # Offense count: 16
 # Configuration parameters: CountComments.
 Metrics/ClassLength:
-  Max: 250
-  Exclude:
-    - 'shoes-core/lib/shoes/slot.rb'
```

Comment:
Not sure why this got changed by the auto-corrector. I'm actually wanting to hold the line at 250 and specifically exempt that one file, so could this get changed back?